### ISSUE TITLE
About the training accuracy

### ISSUE THREAD

**Issue by HsuTzuJen** [auroua/InsightFace_TF#10]
It is still 0.000000 after few epochs(ex in 7 epochs).
Should we use 
acc = tf.reduce_mean(tf.cast(tf.equal(tf.argmax(pred, axis=1), tf.argmax(tf.one_hot(labels, depth=args.num_output), 1)), dtype=tf.int64))
instead of 
acc = tf.reduce_mean(tf.cast(tf.equal(tf.argmax(pred, axis=1), labels_s[i]), dtype=tf.int64))
?

I still do not know how to fix it.

**Comment by auroua** [comment]
This bug have fixed.
